julia/core.py

```diff
@@ -286,7 +289,11 @@ def __init__(self, init_julia=True, jl_init_path=None):
                     "Install PyCall by running the following line:\n"
                     """\tjulia -e 'Pkg.add("PyCall"); Pkg.update()'\n""")
             try:
-                self.call('pyinitialize(C_NULL)')
+                if os.name == "nt":
+                    # on windows C_NULL doesn't mean anything
+                    self.call('pyinitialize()')
```

Comment:
In particular, it looks like the solution might be to have `julia/src/dlload.c` use `GetModuleHandle(NULL)` instead of `GetModuleHandleExW(...)` in `jl_load_dynamic_library_`; this will get the handle of the module for the running executable, which I think is what we need to look up symbols in a globally-loaded `libpython`, rather than the `libjulia` module handle.  (i.e. it seems to be the analogue of the `dlopen(NULL)` that `jl_load_dynamic_library_` is already doing on Mac and Linux systems).

@tkelman, @vtjnash, can you comment?